config-scalecube-configuration-service/src/test/java/io/scalecube/config/service/ApiKey.java

```diff
@@ -0,0 +1,57 @@
+package io.scalecube.config.service;
+
+import java.util.Arrays;
+import java.util.Objects;
+
+public class ApiKey {
+
+  private String APIKey;
```

Comment:
Maybe `apiKey` ?

Reply:
this is the type name in the real production data.